boards/posix/nrf52_bsim/bstests.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2018 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _BSTESTS_ENTRY_H
+#define _BSTESTS_ENTRY_H
+
+#include "bs_types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * TEST HOOKS:
+ *
+ * You may register some of these functions in your testbench
+ *
+ * Note that you can overwrite this function pointers on the fly
+ */
+/*
+ * Will be called with the command line arguments for the testcase.
+ * This is BEFORE any SW has run, and before the HW has been initialized
+ */
+typedef void (*bst_test_args_t)(int, char**);
```

Comment:
This whole file does not really follow the Zephyr coding conventions. You add a lot of typedefs and those lines seem a bit too long to me. Is there a reason for this?

Reply:
All lines are under 80 chars :)
typedefs for functions types seem, luckily, allowed

Reply:
Fine, then it was just my eyes misleading me :)